### ISSUE TITLE
2.2.0 syntax error on Node 5

### ISSUE THREAD

**Issue by ox-michaelradionov** [shaketbaby/directory-named-webpack-plugin#19]
Version 2.2.0 has a syntax error on Node 5.12.0, as destructuring is only supported in Node 6+.

```
.../node_modules/directory-named-webpack-plugin/index.js:8
      01   var {honorPackage: mainFields, exclude, include} = optionsToUse;
      01       ^
      01 
      01 SyntaxError: Unexpected token {
      01     at exports.runInThisContext (vm.js:53:16)
      01     at Module._compile (module.js:387:25)
      01     at Object.Module._extensions..js (module.js:422:10)
      01     at Module.load (module.js:357:32)
      01     at Function.Module._load (module.js:314:12)
      01     at Module.require (module.js:367:17)
      01     at require (internal/module.js:20:19)
      01     at Object.<anonymous> 
```

**Comment by shaketbaby** [comment]
Sorry, have fixed, v2.2.1 released.

**Comment by ox-michaelradionov** [comment]
It's fine. Works in 2.2.1. Thank you!
